wesnoth-tb/bn.po

```diff
@@ -0,0 +1,1293 @@
+#: data/campaigns/Two_Brothers/_main.cfg:39
+msgid "Prose and Story Edits"
+msgstr "গদ্য এবং গল্প সম্পাদনা"
+
+#. [about]
+#: data/campaigns/Two_Brothers/_main.cfg:48
+msgid "Campaign Maintenance"
+msgstr "প্রচার রক্ষণাবেক্ষণ"
+
+#. [about]
+#: data/campaigns/Two_Brothers/_main.cfg:57
+msgid "Artwork and Graphics Design"
+msgstr "শিল্পকর্ম এবং গ্রাফিক্স ডিজাইন"
+
+#. [about]
+#: data/campaigns/Two_Brothers/_main.cfg:73
+msgid "Miscellaneous"
+msgstr "বিবিধ"
+
+#. [entry]
+#: data/campaigns/Two_Brothers/_main.cfg:87
+msgid "And special thanks to everyone else who I forgot to mention."
+msgstr "এবং আমি উল্লেখ করতে ভুলে গেছি এমন প্রত্যেককে বিশেষ ধন্যবাদ।"
+
+#. [scenario]: id=01_Rooting_Out_a_Mage
+#: data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg:5
+msgid "Rooting Out a Mage"
+msgstr "একটি ম্যাজ আউট আউট"
+
+#. [part]
+#. This text is shown with the journey map, showing the village in the
+#. foothills along the route between Aldril and Dan’Tonk.
+#. I'm assuming the beacons either use smoke signals or basic fire signals,
+#. with a few agreed ones to request goods or indicate that surplus is for
+#. sale.
+#: data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg:52
+msgid "The remote freehold of Maghre in the western reaches of the Kingdom of Wesnoth was once a peaceful place, its inhabitants largely unaware of the comings and goings of the wider world. Wars and the rumor of wars touched them not. Even merchants were rarely seen, although caravans between Aldril and Dan’Tonk might send a wagon when called by the beacon tower."
+msgstr "ওয়েসনোথের কিংডমের পশ্চিমাঞ্চলে মাগরে দূরবর্তী ফ্রিহোল্ড একসময় শান্তিপূর্ণ জায়গা ছিল, এর বাসিন্দারা বৃহত্তর বিশ্বের আগমন ও চলার বিষয়ে মূলত অজানা।যুদ্ধ এবং যুদ্ধের গুজব তাদের স্পর্শ করে নি।এমনকি বণিকদের খুব কমই দেখা গিয়েছিল, যদিও বীকন টাওয়ারের ডাকা হলে অ্যালড্রিল এবং ড্যানটোনকের মধ্যে কাফেলাগুলি একটি ওয়াগন পাঠাতে পারে।"
+
+#. [part]
+#: data/campaigns/Two_Brothers/scenarios/01_Rooting_Out_a_Mage.cfg:57
+msgid ""
+"Then came the day that a dark mage settled in the region and began seeking sacrifices for his summonings.\n"
+"\n"
+"Skeletons and zombies killed cattle and fired fields. <i>“Fear and obey Mordak the Mage!”</i> they cried in fell voices as they did their foul deeds. People vanished from isolated farmsteads. Men and women began to fear the night, and their children even the bright day. But the nearest lord was more than a day’s ride distant, and messengers sent to seek his help did not return."
+msgstr ""
+"তারপরে সেই দিনটি এসেছিল যে একটি অন্ধকার ম্যাজ এই অঞ্চলে বসতি স্থাপন করেছিল এবং তার তলব করার জন্য ত্যাগ চাইতে শুরু করে।\n"
+"\n"
+"কঙ্কাল এবং জম্বিগুলি গবাদি পশু এবং গুলি চালানো ক্ষেতকে হত্যা করেছিল।<i> \"ম্যাজকে ভয় এবং মেনে চলুন!লোকেরা বিচ্ছিন্ন খামার থেকে নিখোঁজ হয়েছিল।পুরুষ এবং মহিলা রাত এবং তাদের সন্তানদের এমনকি উজ্জ্বল দিনকে ভয় করতে শুরু করে।তবে নিকটতম প্রভু এক দিনের রাইডের চেয়ে বেশি দূরের ছিলেন এবং তাঁর সাহায্যের সন্ধানের জন্য প্রেরিত বার্তাবাহকরা ফিরে আসেনি।"
```

Comment:
`<i>` is italic markup, a bit like HTML - it needs a corresponding `</i>` somewhere

Reply:
fixed in ae153a789173dceec199454650b4d83022775829.